rcl_yaml_param_parser/src/parser.c

```diff
@@ -1152,22 +1155,25 @@ static rcl_ret_t parse_key(
         if (NULL == parameter_ns) {
           param_name = rcutils_strdup(value, allocator);
           if (NULL == param_name) {
-            return RCL_RET_BAD_ALLOC;
+            ret = RCUTILS_RET_BAD_ALLOC;
+            break;
           }
         } else {
           const size_t params_ns_len = strlen(parameter_ns);
           const size_t param_name_len = strlen(value);
           const size_t tot_len = (params_ns_len + param_name_len + 2U);
 
           if (tot_len > MAX_STRING_SIZE) {
-            RCL_SET_ERROR_MSG_WITH_FORMAT_STRING(
+            RCUTILS_SET_ERROR_MSG_WITH_FORMAT_STRING(
               "The name length exceeds the MAX size %d at line %d", MAX_STRING_SIZE, line_num);
-            return RCL_RET_OK;
+            ret = RCUTILS_RET_ERROR;
```

Comment:
RCUTILS_RET_ERROR -> RCL_RET_OK?

Reply:
Reading it again, I think it was wrong before.

Reply:
Yeah, that's what I thought.